orangecontrib/text/widgets/utils/widgets.py

```diff
@@ -21,7 +21,7 @@ def __init__(self, master=None, attr=None, *args):
 
     def synchronize(self):
         if self.master and self.attr:
-            setattr(self.master, self.attr, self.toPlainText().split('\n'))
+            setattr(self.master, self.attr, self.toPlainText().strip().split('\n'))
```

Comment:
This sets the `self.word_list` to `['']` when empty while the scripting part expects it to be `None`.